app/src/main/java/com/example/umlandowallet/ui/Screen.kt

```diff
@@ -7,5 +7,6 @@ sealed class Screen(val route: String) {
     object ListPeersScreen : Screen("list_peers")
     object ConnectPeerScreen : Screen("connect_peer")
     object OpenChannelScreen : Screen("open_channel")
+    object ListChannelsScreen: Screen("list_channels")
```

Comment:
Small nit, but by convention when an object is extending an interface or another class the colon is separated by a space (`ListChannelsScreen : Screen`) as opposed to when it's used to declare a variable type or function return type (`val age: Int`)

Reply:
Updated in https://github.com/ConorOkus/uMlando-wallet/pull/29/commits/3ad40d6b04f55ae536f0e276c6b89093ebe95299